packages/data/tests/unit/services/navi-formatter-test.ts

```diff
@@ -0,0 +1,89 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+// @ts-ignore
```

Comment:
out of curiosity what is this ignoring?

Reply:
It ignores the next line,
```
 import setupMirage from 'ember-cli-mirage/test-support/setup-mirage'; 
```
which there is no typescript module with types for 'ember-cli-mirage/test-support/setup-mirage';